### PR TITLE
sitzmann2020implicit_2

### PR DESCRIPTION
**Pull request by kraja99**

### CODE REVIEW

File: _summaries/sitzmann2020implicit_2.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
+## SIREN
+The authors propose SIREN as a simple architecture that utilizes sine as a periodic activation function. It is formulated as shown below:
+
+![SIREN Formulation](sitzmann2020implicit_2a.png)
+
+where each phi term represents a layer of the network. SIRENs are shown to converge faster than baseline architectures and have some interesting properties; for example the derivative of a SIREN is also a SIREN (given that the derivative of sine is cosine, which in itself is a phase-shifted sine).
+
+### Basic Image Usecase
+The authors provide an example of trying to parameterize a discrete image. 
+In this case each image defines a dataset D = {(x<sub>i</sub>, f(x<sub>i</sub>))}<sub>i</sub> where x<sub>i</sub> is the pixel coordinate (x<sub>i</sub>, y<sub>i</sub>) and f(x<sub>i</sub>)) is the associated RGB color. Network phi (trying to predict f(x<sub>i</sub>)) is fit using an L2 loss function and trained over the dataset.
```
Comment (aabayomi) [inline _summaries/sitzmann2020implicit_2.md:28]: For this image usecase section, it maybe a good idea to add the example of learning from space which i think was the only experiment they trained a network + showing the pixel-wise MSE table.

File: _summaries/sitzmann2020implicit_2.md [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
+
+### Representing Shapes with Signed Distance Functions
+![Mesh Example](sitzmann2020implicit_2e.png)
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Using sine as a periodic activation function over ReLU in implicit neural representations can improve representation performance and converge faster
```
Comment (aabayomi) [inline _summaries/sitzmann2020implicit_2.md:50]: You may add something similar to, SIREN requires a principled weight initialization scheme to train effectively.
